generator/src/OperatorClassGenerator.php

```diff
@@ -157,17 +163,21 @@ public function createClass(GeneratorDefinition $definition, OperatorDefinition
                 if ($type->javascript) {
                     $namespace->addUseFunction('is_string');
                     $namespace->addUse(Javascript::class);
-                    $constuctor->addBody(<<<PHP
-                    if (is_string(\${$argument->name})) {
-                        \${$argument->name} = new Javascript(\${$argument->name});
+                    $constructor->addBody(<<<PHP
+                    if (is_string(\${$argName})) {
+                        \${$argName} = new Javascript(\${$argName});
                     }
 
                     PHP);
                 }
             }
 
             // Set property from constructor argument
-            $constuctor->addBody('$this->' . $argument->name . ' = $' . $argument->name . ';');
+            $constructor->addBody('$this->' . $argName . ' = $' . $argName . ';');
+        }
+
+        if ($encodeNames !== []) {
+            $class->addConstant('PROPERTIES', $encodeNames);
```

Comment:
Adding a `@var array<string, string|null>` docblock here (in addition to the one added to `OperatorInterface` would resolve some psalm warnings we're getting later.

Reply:
It's added to the `OperatorInterface`, that solved the issues.